discovery-provider/src/queries/health_check.py

```diff
@@ -79,6 +82,40 @@ def play_check():
         sign_response=False
     )
 
+# Health check for latest play stored in the db
+@bp.route("/sol_play_check", methods=["GET"])
+def sol_play_check():
+    """
+       limit: number of latest plays to return
+       max_drift: maximum duration in seconds between `now` and the
+       latest recorded play record to be considered healthy
+    """
+    limit = request.args.get("limit", type=int, default=20)
+    max_drift = request.args.get("max_drift", type=int)
+    redis = redis_connection.get_redis()
+
+    latest_db_sol_plays = get_latest_sol_plays(limit)
+    latest_cached_sol_tx = get_pickled_key(redis, latest_sol_play_tx_key)
+
+    response = {
+        'chain_tx': latest_cached_sol_tx,
+        'db_info': latest_db_sol_plays
+    }
+
+    if latest_db_sol_plays:
+        latest_db_play = latest_db_sol_plays[0]
+        latest_created_at = latest_db_play['created_at']
+        drift = (datetime.now() - latest_created_at).total_seconds()
+
+    # Error if max drift was provided and the drift is greater than max_drift
+    error = max_drift and drift > max_drift
```

Comment:
I think this will error if there is no `latest_db_sol_plays` and we never set `drift`?

Reply:
good catch, will add to conditional